src/kubernetes.cc

```diff
@@ -961,6 +961,38 @@ json::value KubernetesReader::FindTopLevelOwner(
   return FindTopLevelOwner(ns, GetOwner(ns, ref->As<json::Object>()));
 }
 
+bool KubernetesReader::ValidatePodConnectivity() const {
+  try {
+    QueryMaster(std::string(kKubernetesEndpointPath) + "/pods?limit=1");
```

Comment:
Let's make it `(void)QueryMaster(...)`, to make it explicit that we're ignoring the return value on purpose.

Reply:
Done.